tgui/packages/tgui/interfaces/Radio.js

```diff
@@ -86,6 +87,14 @@ export const Radio = (props, context) => {
                   content={`Subspace Tx ${subspace ? 'ON' : 'OFF'}`}
                   onClick={() => act('subspace')} />
               )}
+              {!!honk && (
+                <Button
+                  ml={1}
+                  icon="bullhorn"
+                  selected={useHonk}
+                  content={`${useHonk ? 'HONK!' : 'honk.'}`}
+                  onClick={() => act('honk')} />
```

Comment:
`color={useHonk ? "pink" : null}`

Reply:
also why is this wrapped in a template string? Just do `content={useHonk ? 'HONK!' : 'honk.'}`

Reply:
which part is a template string?

Reply:
o